charts/plgd-hub/README.md

```diff
@@ -106,6 +106,7 @@ global:
 | certificateauthority.restartPolicy | string | `"Always"` | Restart policy for pod |
 | certificateauthority.securityContext | string | `nil` | Security context for pod |
 | certificateauthority.service.annotations | object | `{}` | Annotations for certificate-authority service |
+| certificateauthority.service.crt.extraDnsNames | string | `nil` | Extra DNS name for service certificate |
```

Comment:
Is it type `[]string` ? when I look to bitnami charts I see the param as array string